main.cs

```diff
@@ -333,6 +333,10 @@ public static void LoadRoleClasses()
 
             CustomRolesHelper.DuplicatedRoles = new Dictionary<CustomRoles, Type>
             {
+                { CustomRoles.Pestilence, typeof(PlagueBearer) },
+                { CustomRoles.Death, typeof(SoulCollector) },
+                { CustomRoles.War, typeof(Berserker) },
+                { CustomRoles.Famine, typeof(Baker) },
```

Comment:
Can you please do same code as PlaguBearer and Pestilence (I want to get rid of DuplicateRoles or keep it for Mini only, as it just makes the role more complicated and doesn't automate adding settings)

Reply:
wdym by that? should i get rid of these lines entirely and see what I need to update so that it applies to both roles?